src/chem/Chain.js

```diff
@@ -2,6 +2,15 @@ import * as THREE from 'three';
 import Residue from './Residue';
 import ResidueType from './ResidueType';
 
+/**
+ * Residues in chain are either amino acid either nucleic acid (and water)
+ * There might be some modified/mutated residues, which could not be definite by their name (nucleic or amino); In this
+ * case firstly program definites the chain type (by well-known residues) and then definites modified/mutated residues
+ */
+const CHAIN_TYPE = {
+  INDEFINITE: 0, NUCLEIC: 1, AMINO: 2,
```

Comment:
I'd prefer UNKNOWN, UNDEFINED, or even GENERIC to 'indefinite'

Reply:
Yes, I think, me too. Will correct!